src/style.css

```diff
@@ -0,0 +1,78 @@
+* {
```

Comment:
do not overuse `*`-selector, style the elements you need,
use margin: 0 for body tag